core-ui/src/components/Predefined/Details/ServiceBinding/ServiceBindingData.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import LuigiClient from '@luigi-project/client';
+import { Link } from 'fundamental-react';
+import { DefinitionList } from '../../../../shared/components/DefinitionList/DefinitionList';
+import { useTranslation } from 'react-i18next';
+
+export function ServiceBindingData({ spec, status }) {
+  const { t } = useTranslation();
+  const navigateToInstance = instanceName =>
+    LuigiClient.linkManager()
+      .fromContext('namespace')
+      .navigate(`/btp-instances/details/${instanceName}`);
+
+  const navigateToSecret = secretName =>
```

Comment:
sometimes such secret doesn't exist, and it redirects the user to a non-existing page. Could we somehow distinguish if we should only show a name or a link?